tests/integ/conftest.py

```diff
@@ -21,6 +24,32 @@
     running_on_public_ci,
 )
 
+RUNNING_ON_GH = os.getenv("GITHUB_ACTIONS") == "true"
+RUNNING_ON_JENKINS = "JENKINS_HOME" in os.environ
+
+test_dir = os.path.dirname(__file__)
+test_data_dir = os.path.join(test_dir, "cassettes")
+
+SNOWFLAKE_CREDENTIAL_HEADER_FIELDS = [
+    "Authorization",
+    "x-amz-server-side-encryption-customer-key-MD5",
+    "x-amz-server-side-encryption-customer-key-md5",
+    "x-amz-server-side-encryption-customer-key",
+    "x-amz-server-side-encryption-customer-algorithm",
+    "x-amz-id-2",
+    "x-amz-request-id",
+    "x-amz-version-id",
+]
+
+
+def _process_response_recording(response):
```

Comment:
where do we use _process_response_recording?

Reply:
remove this. double-check this file.

Reply:
reviewed file and removed dead code.